R/set_args.R

```diff
@@ -54,6 +54,23 @@ set_legend_args <- function(object, fun.name, ..., legend.name) {
   
   paramsAll <- set_args(fun.name, list(...))
   
+  if(length(legend.name) > 1){
+    paramsAll_df <- as.data.frame(paramsAll, stringsAsFactors = FALSE)
```

Comment:
So it is inside of `set_legend_args`. I created `get_legend_args` as part of this PR to separate the logic that figures out what should be a `legend.args` list item from what is creating the elements inside that list. `set_args` is being called from `set_legend_args` to get the parameters that are being plotted.